src/data_loading/chernobyl.py

```diff
@@ -0,0 +1,31 @@
+"""
+This module provides helper functions to load land cover data close to chernobyl.
+"""
+
+import geopandas as gpd
+from src.constants import GWS_DATA_DIR
+
+
+def get_polesia_data(variant: str = "biotope") -> gpd.GeoDataFrame:
+    """
+    This function loads the Polesia landcover data from shared workspace on Jasmin.
+    Note that this requires access to Jasmin and the relevant shared workspaces.
+
+    Args:
+        variant (str, optional): either "biotope" or "vegetation".
+            Defaults to "biotope".
+
+    Returns:
+        gpd.GeoDataFrame: shape file data loaded as polygons in data frame
+    """
```

Comment:
Very nice & clear documentation! 

Just an FYI and in the interest of sharing best practices 😉 

As best practice for doc-strings [PEP-257](https://www.python.org/dev/peps/pep-0257/) lists (among other):

- Triple quotes are used even though the string fits on one line. This makes it easy to later expand it.
- There's no blank line either before or after the docstring.
- The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".
- Unless the entire docstring fits on a line, place the closing quotes on a line by themselves. This way, Emacs' fill-paragraph command can be used on it.

Applied to this docstring for example, this would give:

```suggestion
    """Return Polesia landcover data from shared workspace on Jasmin.
    
    Note that this requires access to Jasmin and the relevant shared workspaces.

    Args:
        variant (str, optional): Which variant of  the data to load. Must be 
            either "biotope" or "vegetation". Defaults to "biotope".

    Returns:
        gpd.GeoDataFrame: shape file data loaded as polygons in data frame
    """
```

Reply:
Brilliant, thanks for the feedback! That's a good pointer, will try to change them accordingly.